### PR TITLE
Remove redundant zombie spawn chance config

### LINKED ISSUE #134: Remove baby zombie config options in future MC version (1.21?)
I came across this in the server config file for Neoforge which was in inherited from Forge. 
image

In my opinion, this is not the domain of a modloader API. Rather this zombie villager baby chance should be handled by a separate mod made by someone else that Neoforge loads just like any other mod when in mod folder. There's no reason for this config option to be in Neoforge itself. 

Thoughts?

Code:
https://github.com/neoforged/NeoForge/blob/0974ecc143f449087139a6923aa3f0823dce421d/src/main/java/net/minecraftforge/common/ForgeConfig.java#L26-L27

### PR DESCRIPTION
**Pull request by Su5eD**
### The issue

As discussed in #134, NeoForge includes a couple old, redundant config options for altering the spawn chance of zombies / baby zombies. These have been deemed to be unfit for the loader and are due for removal. Spawn chance should be handled by mods instead.

### The solution

Remove the `zombieBaseSummonChance` and `zombieBabyChance` config options.

Fixes #134

### CODE REVIEW

**Comment by Technici4n** [comment]
Hi, can you update this PR to 20.2?

**Comment by Technici4n** [comment]
Note: you have a formatting issue.

**Comment by Su5eD** [comment]
Should be good to go now

File: src/main/java/net/neoforged/neoforge/common/NeoForgeConfig.java [hunk 1]
```diff
@@ -54,18 +50,6 @@ public static class Server {
                     .worldRestart()
                     .define("fullBoundingBoxLadders", false);
 
-            zombieBaseSummonChance = builder
-                    .comment("Base zombie summoning spawn chance. Allows changing the bonus zombie summoning mechanic.")
-                    .translation("neoforge.configgui.zombieBaseSummonChance")
```
Comment (Technici4n) [inline src/main/java/net/neoforged/neoforge/common/NeoForgeConfig.java:59]: Could you remove these entries from the lang file?

**Review by Technici4n — APPROVED** [review]

**Pull request merged**